pkgs/development/python-modules/ffmpy/default.nix

```
@@ -9,7 +9,7 @@
 
 buildPythonPackage rec {
   pname = "ffmpy";
-  version = "0.3.1";
+  version = "0.3.2";
   format = "setuptools";
```

```suggestion
  pyproject = true;
```
and add `setuptools` to `nativeBuildInputs`